nixos/modules/services/misc/nix-daemon.nix

```
@@ -383,8 +383,15 @@ in
         type = types.bool;
         default = true;
         description = lib.mdDoc ''
-          If enabled (the default), checks for data type mismatches and that Nix
-          can parse the generated nix.conf.
+          If enabled (the default), checks that Nix can parse the generated nix.conf.
```

```suggestion
          If enabled, checks that Nix can parse the generated nix.conf.
```

That is already in the search ui.